ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/OptimizeQueueService.java

```diff
@@ -627,6 +627,11 @@ private BigDecimal evalQuotaRate(TableIdentifier tableId, long currentTime) thro
           latestCostTime = latestCostTime + currentTime - latestTaskHistory.getStartTime();
         }
       }
+
+      if (currentTime - latestStartTime == 0) {
```

Comment:
`evalQuotaRate` should calculate the cost of each attempts for a task.

Reply:
yes, this is out of for loop.
I add the check is order to avoid the dividend  is 0 in next step.